dev/dll/Microsoft.UI.Xaml.vcxproj

```diff
@@ -93,13 +94,13 @@
     <Import Project="..\Scroller\Scroller.vcxitems" Label="Shared" />
     <Import Project="..\ScrollViewer\ScrollViewer.vcxitems" Label="Shared" Condition="$(BuildLeanMuxForTheStoreApp) != 'true'" />
     <Import Project="..\Lights\Lights.vcxitems" Label="Shared" />
-    <Import Project="..\Repeater\Repeater.vcxitems" Label="Shared" />
+    <Import Project="..\Repeater\Repeater.vcxitems" Label="Shared" Condition="$(RepeaterExcluded) != 'true'"/>
```

Comment:
My thinking here was that the .props file would have a list of directory names to be included and then we could use MSBuild wildcard expansion to find all *.vcxitems under that set of directories and then pass that list to `<Import/>` (according to https://github.com/microsoft/msbuild/pull/2014) Import tag should allow an item list to be passed to it.

Does that make sense? I'm not entirely sure if it would work, but that seems a bit more maintainable than a list of conditions that needs to be managed in the vcxproj too.

Reply:
Expansion inside Import does not seem to work. I started a thread with the person who added the feature. If i provide the list inline, it works, but not through an ItemGroup.

Reply:
Unfortunately I learnt that this is not possible in msbuild after a long email thread. Will try to simplify a little more.